### ISSUE TITLE
Incorrect calculation from Kerbin to Kerbin Keostationary orbit?

### ISSUE THREAD

**Issue by Svish** [LoicViennois/KSP-DeltaV-Planner#18]
Tried to enter Kerbin to Kerbin and hoped it would allow me to choose Keostationary orbit or show the total for that. Which it kind of does, but seems like it adds an unnecessary step?

![image](https://github.com/LoicViennois/KSP-DeltaV-Planner/assets/142162/ec5f7ccb-623c-4803-a8ac-cf921b576293)

Does `Transit to Kerbin SOI` take something into account that the other two does not, or should it not have been part of the calculation?

**Comment by LoicViennois** [comment]
When you select "Kerbin to Kerbin", you are shown 3 distinct information, you must combine them to know how much dV you need:

- From Kerbin surface to Kerbin SOI: 3400 + 950 m/s
- From Kerbin surface to Keostationary orbit: 3400 + 1115 m/s

You can notice that in this case there is no "Total" row, so the steps are not added.

I could think of adding two distinct options in the dropdown menu: one for Kerbin SOI, and one for Keostationary orbit. It might help to make things more explicit.

**Comment by Svish** [comment]
What confuses me I guess is that I assumed `TakeOff from Kerbin: 3400 m/s` would be for getting from the launch pad into orbit around Kerbin. And once you're in orbit around Kerbin, the next step would then logically be to change that orbit to a Keostationary one. I don't understand what the extra step in-between there is.

Delta-V map images I've found list `3400 m/s` for Launch Pad to low Kerbin orbit, and `1115 m/s` for low orbit to Keyostationary. An extra `950 m/s` isn't really mentioned anywhere, and I don't understand what that's it's for or what "Transit to Kerbin SOI" even means, when you're already in orbit around Kerbin. 🤔

**Comment by LoicViennois** [comment]
I think it can be clearer if you open the site on desktop. When you hover your mouse over one step, it shows what it corresponds to:

1. "TakeOff from Kerbin" corresponds to Delta-V needed to take-off.
2. "Transit to Kerbin SOI" corresponds to Delta-V needed to go from low orbit to "Kerbin sphere of influence", i.e the orbit where your Periapris is the lowest possible (70 km), and your Apoapsis is at the limit of the Kerbin SOI.
3. "Transit to Keostationary orbit" corresponds to Delta-V needed to raise and circularize your orbit from low to keostationary 

Steps 2 and 3 are two distinct paths, starting from low orbit.

![image](https://github.com/LoicViennois/KSP-DeltaV-Planner/assets/9892489/d6c056dd-73f6-4ab7-98c7-51a481c433b1)

(enjoy my drawing skills 😉 )

To summarize, you need:

- 3400 + 950 Delta-V to take off, go into low orbit, then raise your Apoapsis to the limit of the Kerbin SOI.
- 3400 + 1115 Delta-V to take off, go into low orbit, then raise your Apoapsis to keostationary orbit then circularize.

Hope this helps.

**Comment by Svish** [comment]
Aah, I guess it's "confused" from me selecting Kerbin as both source and target, and I guess both Kerbin SOI _and_ Keostationary matches Kerbin as a target, so it shows both.

That makes sense.

I'll just close this then, as the issue probably is mostly me "abusing" the calculator by putting the same body as source and target.

**Comment by LoicViennois** [comment]
You are not 'abusing' the calculator 😉 , it is the expected behavior, when you select Kerbin as both source and target it shows you the Kerbin-related possibilities, but for better user experience I should have added a dedicated option in the dropdown menu. I will think about it
